extensions/wikia/Search/js/WikiaSearch.js

```diff
@@ -81,8 +89,103 @@ var WikiaSearch = {
 			searchForm.submit();
 		});
 
+	},
+	trackSearchResultClick: function(clickedElement) {
+		var queryparams = new URL(window.location).searchParams;
```

Comment:
IE 11 will explode here until we have some polyfill

consider `url` package, it's used eg. in `auth-pages`.

can't you go with `window.location.search`?

Reply:
then I'd need to parse manually whole query string. Do we even support IE11?

Reply:
@xkxd checked how many sessions we have from IE users, and all of IE versions are getting us 1,52% of sessions (data for the last month). @szczeles did the same check in DW and looks like IE 11 itself is getting us 0.8%. So we both think there is no big deal with broken tracking for this browser